readthedocs/api/v2/proxied_urls.py

```diff
@@ -15,7 +15,7 @@
 api_footer_urls = [
     url(r'footer_html/', ProxiedFooterHTML.as_view(), name='footer_html'),
     url(r'search/$', ProxiedPageSearchAPIView.as_view(), name='search_api'),
-    url(r'embed/', ProxiedEmbedAPI.as_view(), name='embed_api'),
+    url(r'embed/', ProxiedEmbedAPI.as_view(), name='api_embed'),
```

Comment:
this needs to match the name from the "normal" api. I can rename the other one instead if we want.